pycbc/workflow/core.py

```diff
@@ -197,18 +197,26 @@ def __init__(self, cp, name,
 
         if exe_url.scheme in ['', 'file']:
             if os.path.isfile(exe_url.path):
-                self.add_pfn(exe_path)
+                self.add_pfn(exe_path,site='local')
```

Comment:
Done in https://github.com/ligo-cbc/pycbc/pull/1194/commits/fa69679765ab5562bcef9eb52c0fc83615fe00e4